### ISSUE TITLE
Error in log connecting

### ISSUE THREAD

**Issue by callapple** [thetatoken/guardian-mainnet-guide#102]
I see this error in the log repeatedly on my machine...what is going on?

[2021-04-08 16:47:07.728] [info] [ThetaEdgeLauncher] time="2021-04-08T16:47:07+09:00" level=warning msg="client.Call res: <nil>, error: Post \"http://127.0.0.1:7935/rpc\": dial tcp 127.0.0.1:7935: connect: connection refused" prefix=launcher

**Comment by Lionel2298** [comment]
What version of GN are you running on ?

**Comment by Lionel2298** [comment]
What version of GN are you running on ?

**Comment by craigshimizu** [comment]
Version 2.2.10 (2.2.10)

**Comment by Lionel2298** [comment]
@craigshimizu whats your theta address so I can run a check for you

**Comment by craigshimizu** [comment]
I have two. This is one of them:  0xcf30b32617f36856617f844bc18753daac95d168

**Comment by Lionel2298** [comment]
Okay. Which is the second address ?

**Comment by craigshimizu** [comment]
What did you find?

**Comment by Lionel2298** [comment]
I can see that you GN is currently unstaked.
I can also see that you sent tfuel to your theta GN address.
I assume that was a mistake ?

**Comment by craigshimizu** [comment]
What is okay?

**Comment by craigshimizu** [comment]
Probably. What should I do?

**Comment by Lionel2298** [comment]
What’s the second theta address ?

**Comment by craigshimizu** [comment]
Please let me know what you found. Or is there someone else I can contact?

**Comment by Lionel2298** [comment]
I told you what I found already. I can see that you GN is currently unstaked.
I can also see that you sent tfuel to your theta GN address.
I assume that was a mistake ?

**Comment by adubkov** [comment]
I'm having same issue. Just installed edge node (2.3.62) on mac. Does not display Watch. Having loop of same errors in logs.

**Comment by Lionel2298** [comment]
What version of MacOS are you running on ?

**Comment by adubkov** [comment]
11.2.3

**Comment by Lionel2298** [comment]
I guess that’s the Big Sur OS. You can contact the Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by adubkov** [comment]
I actually tried. Telegram support is useless. They asked me to check my internet.... and then just spammers reply me in DM asking to "recover you wallet on our fake website"

**Comment by adubkov** [comment]
Actually It was different channel: https://t.me/thetanetworkglobal

**Comment by Lionel2298** [comment]
Just sent you the official now.

**Comment by adubkov** [comment]
Yeah, and guy on that "official" just asked me to unlock my walled on "https://www.connectingthetawallet.org". Sure.... 
Also your account seems "official" too.
